xz/FeatureModel.xml

```diff
@@ -2095,8 +2246,8 @@
     <constraint req="all" exprKind="pos"> NoThreads * Threads_val = 1</constraint>
     <constraint req="all" exprKind="pos"> NoBytesCompr * BytesCompr_val = 1</constraint>
     <constraint req="all" exprKind="pos"> NoPercentCompr * PercentCompr_val = 1</constraint>
-    <constraint req="all" exprKind="pos"> NoSize * Size_val = 1</constraint>
-    <constraint req="all" exprKind="pos"> NoFlushTO * FlushTO_val = 1</constraint>
+    <constraint req="none" exprKind="pos"> (!NoSize * Size_val != 0) | (NoSize * Size_val = 0)</constraint>
```

Comment:
In the code base of xz, the feature variable of Size is 0 iff the feature Size is not selected. So my intention was to express this equivalence with the constraint. I was not sure if there is a respective operator for equivalences so I tried it that way.

Reply:
How about:
`<constraint req="all" exprKind="pos"> NoSize * Size_val = 0</constraint>`
If `Size` is selected in a configuration (i.e., `NoSize` has the value 0), the value of `Size_val` doesn't matter.
If `Size` is deselected (i.e., `NoSize, has the value 1)` `Size_val` has to be 0.
Would this express what you want?

Reply:
I see.
I have the following suggestion to express what you want:
` <constraint req="all" exprKind="pos">Size_val &gt;= Size</constraint>`
`<constraint req="all" exprKind="pos">NoSize * Size_val = 0</constraint>`
Would this work?

Reply:
I am not sure what the operator `>=` expresses, but I guess inequality. For my understanding, the constraint
`<constraint req="all" exprKind="pos">Size_val &gt;= Size</constraint>`
would be true if `Size` is not selected. If `Size` is selected, so it has the value 1, `Size_val` can have any value but 1. That's not quite what I want to express since it can have any value but 0. 

But maybe the constraints
`<constraint req="all" exprKind="pos">Size * Size_val &gt;= 0</constraint>`
`<constraint req="all" exprKind="pos">NoSize * Size_val = 0</constraint>`
could work?

Reply:
> `<constraint req="all" exprKind="pos">Size * Size_val &gt;= 0</constraint>`

What would this do? Does this have any effect? `>= 0`  would still permit all values.

> `Size_val` can have any value but 1.

No; any value but 0. 

> <constraint req="all" exprKind="pos">Size_val &gt;= Size</constraint>

This constraint can be translated as: If `Size` is `1` (selected), `Size_val` has to greater or equal 1. But this constraint is only here to cover for the case when `Size` is selected.

To ultimately force `Size_val` to be `0` when Size is deselected, I use the second constraint:

> <constraint req="all" exprKind="pos">NoSize * Size_val = 0</constraint>

@Keller-1998 If this is still unclear, we can also schedule a brief meeting :-)